panedr/panedr/__init__.py

```diff
@@ -4,4 +4,4 @@
 __version__ = pbr.version.VersionInfo('panedr').release_string()
 del pbr
 
-from .panedr import *
+from panedrlite import *
```

Comment:
Could we make this an explicit import instead of `*`?

Reply:
I have now changed these to explicit imports. However, a `__all__` is defined in panedr.py, so this currently does the same thing twice, in two different locations. This might make things confusing for future developers, so I think only one of the two should happen. Should I remove the `__all__` line from panedr.py?